docs/resources/project.md

```diff
@@ -128,3 +128,9 @@ Required:
 Optional:
 
 - **description** (String)
+
+## Note
+### Repository assignment & unassignment Limitations
+- Project provider has limitations while assigning or de-assigning large number of repositories through repos attribute. With large number of repos you might observe system disruptions and all the repositories might not get assigned/unassigned to/from desired project.
```

Comment:
"unassign"

Reply:
This is just regular paragraph. No need to be bullet point.

Reply:
Done